cypress/e2e/pages/auth/login.cy.ts

```diff
@@ -0,0 +1,20 @@
+/// <reference types="cypress" />
+
+describe('Login Page', () => {
+  beforeEach(() => {
+    cy.visit('/login');
```

Comment:
Not super familiar with sequence in which tests occur in Cypress when there are multiple tests but I could imagine that if this test is run after another test that requires login it would fail because we aren't logging the user out first before the test. Should we be ensuring that the user is logged out first here with a `cy.visit('/logout')`?